src/client/content_mapblock.cpp

```diff
@@ -1616,22 +1627,23 @@ void MapblockMeshGenerator::drawNode()
 /*
 	TODO: Fix alpha blending for special nodes
 	Currently only the last element rendered is blended correct
+	^ TODO: outdated info?
```

Comment:
I think the entire comment doesn’t belong here.

Reply:
Removed the whole comment now.